packages/@ourworldindata/grapher/src/modal/DownloadModal.scss

```diff
@@ -128,18 +100,17 @@
 }
 
 .static-exports-options {
-    margin: 1em 0 1.5em;
-    padding-bottom: 1.5em;
+    margin: 16px 0 24px;
+    padding-bottom: 24px;
     border-bottom: 1px solid #f2f2f2;
 
     > .checkbox + .checkbox {
         margin-top: 8px;
     }
 }
 
-&.GrapherComponentNarrow .DownloadModalContent {
-    .grouped-menu-icon + .grouped-menu-content {
-        padding-left: 1em;
-        padding-right: 0.5em;
+&.GrapherComponentNarrow .download-modal-content {
+    .grouped-menu-icon img {
+        display: none;
     }
 }
```

Comment:
This is where the image is hidden :)

Reply:
Ah, right! My viewport just wasn't narrow enough ;)